extensions/ui-extension/src/fields/PurchaseOrder.tsx

```diff
@@ -19,24 +19,36 @@ export function PurchaseOrderInput() {
    const paymentType = selectedPaymentOption[0].type;
 
    const [isPurchaseOrder, setIsPurchaseOrder] = useState(false);
+   const [hasAttribute, setHasAttribute] = useState(false);
 
    const applyAttributeChange = useApplyAttributeChange();
-   const applyPurchaseOrderChange = (value: string) =>
-      applyAttributeChange({
-         key: 'Purchase Order',
-         type: 'updateAttribute',
-         value,
-      });
+   const applyPurchaseOrderChange = (value: string) => {
+      try {
+         applyAttributeChange({
+            key: 'Purchase Order',
+            type: 'updateAttribute',
+            value,
+         });
+         setHasAttribute(true);
+      } catch (error) {
+         if (paymentType == 'manualPayment') {
+            console.error(error);
+         }
+      }
```

Comment:
Let's unconditionally report the error just in case

```suggestion
      } catch (error) {
         console.error(error);
      }
```